packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -25,6 +25,23 @@ const displayAnalytics = request => {
   return request.path.startsWith('/buy')
 }
 
+const skipPage = async request => {
+  const analytics = await request.cache().helpers.analytics.get()
+
+  if (
+    analytics &&
+    analytics[ANALYTICS.pageSkipped] !== true &&
+    analytics[ANALYTICS.seenMessage] &&
+    analytics[ANALYTICS.skipPage] !== true
+  ) {
+    await request.cache().helpers.analytics.set({ [ANALYTICS.skipPage]: true, [ANALYTICS.pageSkipped]: true })
+    return true
```

Comment:
I don't think you're testing the return value here (or underneath). Try removing them and see if your tests all still pass. If they do, you need to add tests for the return value.